### ISSUE TITLE
UnboundLocalError: local variable 'process' referenced before assignment

### ISSUE THREAD

**Issue by olgabot** [manubot/manubot#165]
Just created a repo this morning and followed the [SETUP](https://github.com/manubot/rootstock/blob/master/SETUP.md) instructions, including AppVeyorBot. But, the build is currently failing for master: https://travis-ci.com/czbiohub/primate-brain-organoid-paper/builds/134583403 (full deploy error message in details below) and the GitHub Pages HTML isn't loading: https://czbiohub.github.io/primate-brain-organoid-paper/

Is there a configuration step that I missed? There *was* an issue with setting the `MANUBOT_SSH_PRIVATE_KEY` (I didn't hit `Add` 🤦‍♀ ) but it is now set. Do you have any suggestions?

### Deploy error message

<details>

```
Deploying application
+set -o errexit -o nounset -o pipefail
++dirname czbiohub/primate-brain-organoid-paper
+OWNER_NAME=czbiohub
++basename czbiohub/primate-brain-organoid-paper
+REPO_NAME=primate-brain-organoid-paper
+export OWNER_NAME REPO_NAME
+envsubst
+mv webpage/README-complete.md webpage/README.md
+git config --global push.default simple
++git log --max-count=1 --format=%ae
+git config --global user.email daniel.himmelstein@gmail.com
++git log --max-count=1 --format=%an
+git config --global user.name 'Daniel Himmelstein'
+git checkout master
Previous HEAD position was a121e47... Update environment on 2019-10-31
Switched to branch 'master'
M	webpage/README.md
Your branch is up to date with 'origin/master'.
+git remote set-url origin git@github.com:czbiohub/primate-brain-organoid-paper.git
++ssh-agent -s
+eval 'SSH_AUTH_SOCK=/tmp/ssh-hEKmZD0aBpID/agent.8053; export SSH_AUTH_SOCK;
SSH_AGENT_PID=8054; export SSH_AGENT_PID;
echo Agent pid 8054;'
++SSH_AUTH_SOCK=/tmp/ssh-hEKmZD0aBpID/agent.8053
++export SSH_AUTH_SOCK
++SSH_AGENT_PID=8054
++export SSH_AGENT_PID
++echo Agent pid 8054
Agent pid 8054
+set +o xtrace
Identity added: (stdin) ((stdin))
+git remote set-branches --add origin gh-pages output
+git fetch origin gh-pages:gh-pages output:output
Warning: Permanently added the RSA host key for IP address '140.82.114.4' to the list of known hosts.
From github.com:czbiohub/primate-brain-organoid-paper
 * [new branch]      gh-pages   -> gh-pages
 * [new branch]      output     -> output
 * [new branch]      gh-pages   -> origin/gh-pages
 * [new branch]      output     -> origin/output
+manubot webpage --timestamp --no-ots-cache --checkout=gh-pages --version=a121e479506eb0232a418bb2ac5b05ebe291ab34
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/manubot/bin/manubot", line 8, in <module>
    sys.exit(main())
  File "/home/travis/miniconda/envs/manubot/lib/python3.7/site-packages/manubot/command.py", line 185, in main
    function(args)
  File "/home/travis/miniconda/envs/manubot/lib/python3.7/site-packages/manubot/webpage/webpage_command.py", line 15, in cli_webpage
    configure_args(args)
  File "/home/travis/miniconda/envs/manubot/lib/python3.7/site-packages/manubot/webpage/webpage_command.py", line 54, in configure_args
    checkout_existing_versions(args)
  File "/home/travis/miniconda/envs/manubot/lib/python3.7/site-packages/manubot/webpage/webpage_command.py", line 111, in checkout_existing_versions
    f"Attempting checkout with the following command:\n{shlex_join(process.args)}"
UnboundLocalError: local variable 'process' referenced before assignment
Script failed with status 1
failed to deploy
```

</details>

**Comment by dhimmel** [comment]
Oh my! This is [also failing](https://travis-ci.com/manubot/rootstock/builds/134485780#L2444) on `rootstock`. Will get a fix out ASAP.

**Comment by dhimmel** [comment]
Thanks @olgabot for pointing this out. After we merge https://github.com/manubot/manubot/pull/166, I'll update rootstock to use the updated `manubot` Python package. You'll then have to update your rootstock version according to [these instructions](https://github.com/manubot/rootstock/blob/a121e479506eb0232a418bb2ac5b05ebe291ab34/SETUP.md#merging-upstream-rootstock-changes).

Also I'll look into how we didn't notice this issue, since ideally we'd get notified of this without a user having to experience the failure!

**Comment by dhimmel** [comment]
Okay https://github.com/manubot/rootstock/commit/7cf9071212ce33116ad09cf2237a370b180a3c35 fixed this issue and [CI deployment now works again](https://travis-ci.com/manubot/rootstock/builds/134602372#L2392).

I opened https://github.com/czbiohub/primate-brain-organoid-paper/pull/2 to fix the issue for @olgabot's manuscript.

A little bit on what happened to enable a production bug like this:

1. `manubot/manubot` didn't fully test the `manubot webpage` command. Therefore this bug was not caught by CI of this repo, although an integration test would have likely caught it. Todo: better `manubot webpage` tests.

2. The bug wasn't detected by the CI runs on https://github.com/manubot/rootstock/pull/274 because pull request builds cannot test deployment. However, once we merged that PR, the `master` CI build did fail at the deploy stage. I did not get a notification. Todo: enable email notifications for `master` branch failures. This would hopefully allow us to hotfix a deployment issue before any users are affected.

Thanks @olgabot for reporting this quickly and the PR! Will work on avoiding similar issues in the future.
